### PR TITLE
nzpmc-093/Delete-answers-and-questions

### PR DESCRIPTION
**Pull request by BW82262**
**Issue**

Lack of mutations to delete answer options.

**Solution**

Added the respective mutations to carry out the operation.

**Reviewers**

@UoaWDCC/team-nzpmc

### CODE REVIEW

**Review by lanxdrey19 — APPROVED** [review]
LGTM

File: src/controllers/option.ts [hunk 1]
```diff
@@ -106,9 +106,36 @@ const editQuestionOption = async (
...
+
+        const question = await quiz.questions.findById(questionID)
+        if (!question || !question.options) {
+            throw new NotFoundError()
+        }
+
+        const option = await question.options.findById(optionID)
+        if (!option) {
+            throw new NotFoundError()
+        }
```
Comment (scorpionknifes) [inline src/controllers/option.ts:128]:
```suggestion
```

File: src/controllers/option.ts [hunk 2]
```diff
@@ -106,9 +106,36 @@ const editQuestionOption = async (
...
+
+        return question.options.delete(optionID)
+        // return quiz.questions.delete(questionID)
```
Comment (scorpionknifes) [inline src/controllers/option.ts:131]:
```suggestion
```

**Review by scorpionknifes — APPROVED** [review]

**Comment by scorpionknifes** [comment]
Do we want to prevent authors from deleting options when quiz is in progress?

**Pull request merged**